pyhtools/attackers/Network/arpspoofer.py

```diff
@@ -5,7 +5,7 @@
 
 
 import scapy.all as sp
-import argparse
+import argparse, os
```

Comment:
Let's avoid importing `os` module here since we only require it while running it inside main block.

Instead let's import `os` inside the main block.